glossary.md

```diff
@@ -0,0 +1,64 @@
+# Glossary
+
+This document defines some terms that are used across this specification.
+
+<details>
+<summary>Table of Contents</summary>
+
+<!-- toc -->
+
+- [User Roles](#user-roles)
+  * [Application Author](#application-author)
+  * [Provider Author](#provider-author)
+  * [Integration Author](#integration-author)
+  * [Library Author](#library-author)
+- [Common](#common)
+  * [Feature Flag API](#feature-flag-api)
+  * [Feature Flag SDK](#feature-flag-sdk)
+  * [Provider](#provider)
+  * [Integration](#integration)
+  * [Context](#context)
+
+<!-- tocstop -->
+
+</details>
+
+## User Roles
+
+### Application Author
+
+The maintainer of an an application or service which utilizes the feature flags API to manage functionality.
+
+### Provider Author
+
+The maintainer of an SDK-compliant provider which implements the necessary interfaces required for flag evaluation.
+
+### Integration Author
+
+The maintainer of an SDK-compliant integration which implements additional secondary functionality besides flag evaluation.
+
+### Library Author
+
+The maintainer of a shared library which is a dependency of many applications or libraries, which utilizes the feature flags API to allow consumers to manage library functionality.
+
+## Common
+
+### Feature Flag API
```

Comment:
Feature Flag API or OpenFeature API?

Reply:
This was something I went back and forth on. I really don't have a strong opinion either way. Do you?